guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -66,3 +66,9 @@ For BIOS provisioning, you must associate a PXELinux template with the operating
 For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
 If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
 endif::[]
+
+.Bonded network interfaces
+You can configure a bonded interface to be used during the installation process, such as for downloading installation content, and for the provisioned system to be used after the provisioning process is finished.
+{Project} cannot PXE boot a bonded interface that requires configuration on a network switch as well as on your host.
+Once your host loads a kernel, either the kernel of an installer or an operating system, bonding works as expected.
```

Comment:
```suggestion
Bonding will work correctly once the host starts up with a kernel, whether it is from an installer or an operating system.
```

Reply:
I'm not going to use this suggestion because it introduces a future tense, a phrasal verb and pronoun, which all are undesirable for our style. However, I think that we shouldn't use "once" either.
```suggestion
When your host loads a kernel, either the kernel of an installer or an operating system, bonding works as expected.
```